README.md

```diff
@@ -121,7 +121,7 @@ Like all other commands, you can specify different providers like `k8s`, `aws`,
 
 ![](docs/images/cnquery-scan.gif)
 
-These files can also contain multiple query packs for many different target systems. For an example, see `examples/multi-target.mql.yaml`.
```

Comment:
Is this file gone now?

Reply:
Yep. And I couldn't find a replacement that does what this describes.

Reply:
@chris-rock should we nuke this or make a new example file?